Tests/XCUITests/TabsPerformanceTests.swift

```diff
@@ -63,6 +63,8 @@ class TabsPerformanceTest: BaseTestCase {
         app.launch()
         waitForTabsButton()
         waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.tabsButton].staticTexts["20"])
+        waitUntilPageLoad()
```

Comment:
this is only for website to load not for the homescreen...

Reply:
When the browser is launched after the database is loaded, the page on the tab is shown instead of the homescreen.
![Simulator Screenshot - iPhone 14 - 2023-06-12 at 11 47 41](https://github.com/mozilla-mobile/firefox-ios/assets/1740517/aa7de327-ad80-428f-b1e2-e64a3510f211)
But I guess a `sleep(5)` is sufficient.